### PR TITLE
Add Clickhouse mixin

### PR DESCRIPTION
**Pull request by JonathanWamsley**
This PR adds the Clickhouse mixin featuring 3 dashboards with an optional logs error panel and a prometheus alert.

### CODE REVIEW

File: clickhouse-mixin/Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+dashboards_out: mixin.libsonnet config.libsonnet $(wildcard dashboards/*)
+	@mkdir -p dashboards_out
+	mixtool generate dashboards mixin.libsonnet -d dashboards_out
+
+prometheus_alerts.yaml: mixin.libsonnet alerts/*.libsonnet
+	mixtool generate alerts mixin.libsonnet -a prometheus_alerts.yaml
+
+
+.PHONY: deploy deploy_rules deploy_dashboards
+deploy: deploy_rules deploy_dashboards
```
Comment (BinaryFissionGames) [inline clickhouse-mixin/Makefile:34]: I think this rule should be removed, the rules it depends on doesn't exist

Comment (JonathanWamsley) [reply clickhouse-mixin/Makefile]: Removed rule

File: clickhouse-mixin/dashboards/clickhouse-latency.libsonnet [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+        graphTooltip='shared_crosshair',
+        uid=dashboardUid,
+      )
+      .addLink(grafana.link.dashboards(
+        asDropdown=false,
+        title='Other clickhouse dashboards',
+        includeVars=true,
+        keepTime=true,
+        tags=($._config.dashboardTags),
+      )).addTemplates(
```
Comment (rgeyer) [inline clickhouse-mixin/dashboards/clickhouse-latency.libsonnet:471]: 👌

**Review by rgeyer — APPROVED** [review]
Looks good! Great work

**Pull request merged**